### ISSUE TITLE
Please finish the print run!

### ISSUE THREAD

**Issue by MineRobber9000** [tildetown/zine#34]
It's been months, and I wonder if will the zine be finished anytime soon? I really want to at least see this issue released, even if we don't make an issue 3.

I'm looking at you, @jumblesale.

**Comment by insom** [comment]
Is there anything other townies can do to help out? I can handle some North American print runs and postage maybe if we have enough things together for another issue? I haven't been keeping track of things, I'm afraid.

**Comment by m455** [comment]
I can't do much money-wise,but if there is anything I can help out with I can help

**Comment by MineRobber9000** [comment]
First, we should try to reach out to @jumblesale, and see what's taking them so long. If they forgot about it and needed us to kick them in the butt to remember, so be it. I personally believe we have enough for a print run, although we probably should formalize when we should begin the process for print runs. (see #33 ) If there are issues on @jumblesale's end (financial, time, etc.) then @insom can work on the print run.

Until further notice, this issue is closed. (seeing as how we're waiting on a response) Let us know if you get a response from @jumblesale.

**Comment by fonorobert** [comment]
I can handle some printing and postage in Europe if that helps

**Comment by resir014** [comment]
@MineRobber9000 Has anyone ever gotten in touch with @jumblesale? I haven't been on that much on the server, but I haven't seen him on every time I go online.

**Comment by MineRobber9000** [comment]
We're trying to reach out to him and get a response at the moment.

**Comment by jumblesale** [comment]
hey sorry I wasn't getting the notification for this. I've got the layout which I will upload when I'm at my PC. if anyone could print and ship it that would be so amazing. I really can't afford to do it any time soon. I'll get the pdf uploaded asap xx

**Comment by MineRobber9000** [comment]
Alright @jumblesale, thanks for reporting in. Pinging @insom @fonorobert @6455

**Comment by fonorobert** [comment]
I'm game, though i don't know how the previous issue was done, but i can definitely do a print and mail it out to European townies.

**Comment by MineRobber9000** [comment]
@fonorobert Basically, @jumblesale will upload the layout (layout.pages/layout.pdf, see "issue_1") and all you have to do is print it out, add a cover (your choice of style) and mail it out. We should make a mailing address database somewhere so we can get addresses. The zine delivery could've been handled much better but this is just a fledgeling attempt at something bigger.

**Comment by jumblesale** [comment]
last time I went round irc asking everyone for their addresses, then I printed it, stapled it and posted it off to everyone. some people created a `.address` in their ~ directory but I'm not sure if that really caught on

**Comment by fonorobert** [comment]
ok, we can put the call out and then go with whatever number of folks want a physical copy

**Comment by MineRobber9000** [comment]
sure! @insom can handle US and @fonorobert can handle European requests. waiting for @insom's confirmation that they can handle US requests.

**Comment by mejoemejoe** [comment]
I also have the ability to make zines here in the US. Please consider me for this project.

**Comment by jumblesale** [comment]
joe please do! all we need to do is collect addresses from US townies then we are ready to go :)

**Comment by fonorobert** [comment]
fyi, i made an announcement in bbj about printing and that folks should put their addresses in .address. also, yay @mejoemejoe :)

**Comment by insom** [comment]
I'm actually not in the US (Hi, 🇨🇦), I was going to cover North America, but if @mejoemejoe can handle some of the US run that would be really helpful, I have no idea what cross border postage costs.

Maybe once people have added stuff to `~/.address` I'll put together a Google Document and we can mark off which townies we're in a position to print and post to?

**Comment by fonorobert** [comment]
Sounds good, although i'm sure everyone would be happy to receive two prints, one from you and one from Joe 😄

**Comment by mejoemejoe** [comment]
I will be happy to handle the printing, stapling, sorting and mailing of zines. I would definitely appreciate some assistance with postage, although I don't mind offsetting the costs to send stuff off to your lovely homes and businesses :love_letter: 

I will hand-letter your names and addresses in Chancery Hand, too! Just like the letters I send to @jumblesale. 

@insom, I think a Google Doc is a great idea, especially for townies who don't log in as much as they should. Please add me (joe@daelis.com) so that way I can see whom to send things to. 

If any of you have any ideas, just let me know and we can iron out the details.

![bitmoji](https://render.bitstrips.com/v2/cpanel/10178725-234496437_91-s1-v1.png?transparent=1&palette=1&width=246)

**Comment by MineRobber9000** [comment]
The way I see it, we should make a spreadsheet. @jumblesale can add to the spreadsheet the townies whose addresses they have and then we can add people with .address files to the list (~/.address supersedes previous list) and mark down who can ship to who.

**Comment by mejoemejoe** [comment]
Let's not burden poor @jumblesale any more than he is now. I'll make a Google forms and have people sign up that way.

**Comment by jumblesale** [comment]
there's a script in my ~ under Code which will get all .address files
together x

**Comment by MineRobber9000** [comment]
Suggestion: (someone should talk to vilmibm about this, I almost always miss them when they're on IRC) possibly non-critical extra town funds could be used to reimburse the people who send out the zine (I'm imagining a tilde.town treasurer who deals with these requests bc that way we don't have to bother vilmibm every time an issue is sent)

**Comment by fonorobert** [comment]
@MineRobber9000 i'd rather cover it myself. i don't want to put pressure on anyone else to do the same though if it turns out to be more cost than expected or sg.
tbh for me personally this is a fun opportunity to put something back into the community and it feels good to do that with my own resources (ie. printer at my mom's office 😆 )

i also wanna do the addresses in handwriting, but my handwriting is p bad so the zines might get lost in the mail 🙈

**Comment by m455** [comment]
@fonorobert don't worry about your handwriting. my hand writing is horrible, but if I write REALLY slowly I can make it legible enough haha. also I mentioned to @insom that, if the costs are low enough i'll help with postage costs and printing if I can afford it. kinda broke right now. I also want to contribute to tilde.town as it's been a great privilege to have access to so many things.

**Comment by jumblesale** [comment]
@6455 you have contributed so much towards making tilde.town the wonderful place it is
